packages/xo-server/src/api/xosan.js

```diff
@@ -861,6 +861,7 @@ async function umountDisk(localEndpoint, diskMountPoint) {
 }
 
 async function createNewDisk(xapi, sr, vm, diskSize) {
+  // VDI size must be between 1 MB and 2088960 MB
```

Comment:
By any chances, would there be any sources we could link to?

Reply:
This is a log from XAPI/SM. Funny story: it was `VDI Invalid size [opterr=VDI size cannot exceed 2093051 MB]` on XenServer 7.1.

It changed in 8.1. I'm not able to find this string in the source using GitHub search.

Reply:
https://github.com/xapi-project/sm/blame/42a084917a64c63cb921f258b638d8342c147c3f/drivers/vhdutil.py#L31

Reply:
Thanks! Github search is the worst :disappointed:

Reply:
@olivierlambert I searched locally, then on github, but I had a hunch as to where it would be.

Reply:
@julien-f I don't think it's worth a changelog, because we are the only ones to see the issue, because there is yet another compounding problem in python.